src/components/views/rooms/SendMessageComposer.tsx

```diff
@@ -74,11 +76,25 @@ function addReplyToMessageContent(
     }
 }
 
+function addAttachmentToMessageContent(
+    content: IContent,
+    attachEventId: string,
+): void {
+    const attachContent = {
+        'm.relates_to': {
+            'rel_type': 'org.matrix.msc2881.m.attachment',
```

Comment:
throughout we generally prefer to use `UnstableValue` from the js-sdk for identifiers like this. It makes for transitioning a bit easier, particularly when/if the MSC becomes stable.

There should be some example usages in the code somewhere: look for `new UnstableValue`